submissions/SerhiiYakovenko/memory-pair-game/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+  padding: 0;
+  color: black;
+  box-sizing: border-box;
+}
+
+body {
+  margin: 10px 0;
+}
+
+.boardGrid {
+  max-width: 960px;
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+}
+
+.card {
+  position: relative;
+  transition: all 0.4s linear;
+  transform-style: preserve-3d;
+  margin: 5px;
+}
+
+.card,
+.back,
+.front {
+  height: 130px;
+  width: 190px;
+}
+
+.back,
+.front {
+  position: absolute;
+  backface-visibility: hidden;
+}
+
+.front {
+  z-index: 2;
+  background: url("./img/front.jpeg") no-repeat center center / contain;
+}
+
+.back {
+  transform: rotateY(180deg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center center;
+}
+
+.selected {
+  transform: rotateY(180deg);
+}
+
+.match .front {
+  background: url("./img/favicon.png") no-repeat center center / contain !important;
```

Comment:
!important ????????? 
Why are you using this ? 
try to implement without important

Reply:
Frankly speaking, I don't know why it's here, as it works well without it, seems I had some issues with its display before and didn't make double-check of redundant things. 
Thanks, for noticing.